### DISCUSSION TITLE
TaoX11 and vcpkg

### DISCUSSION THREAD

**Discussion by pierrebizz** [RemedyIT/taox11#401] [Q&A]
My current project pulls ACE/TAO from vcpkg, and I'm always a little behind on the latest version.

How can I proceed to build TaoX11 against a version of ACE/TAO that is not the latest, as the bootstrap process pulls the latest ACE/TAO repository.

Also, how can I build TaoX11 to compile using ACE/TAO static libraries ?

**Comment by jwillemsen** [comment]
The version of ACE/TAO/MPC is configured in the `etc/brix11rc` file (see https://github.com/RemedyIT/axcioma/blob/master/etc/brix11rc). When we make a release we fix the ACE/TAO/MPC version to the tested version.

We haven't tested TAOX11 with static libraries, brix11 has currently no support for it. Our focus has been the shared library configuration as this is mandatory for ciaox11/dancex11. Support for static could very likely be added, please contact `sales@remedy.nl` to discuss this further.
